specification/storagesync/resource-manager/readme.md

```diff
@@ -26,18 +26,27 @@ These are the global settings for the Storage Sync API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-03-01
+tag: package-2019-06
 ```
 
 
+### Tag: package-2019-06
+
+These settings apply only when `--tag=package-2019-06` is specified on the command line.
+
+```yaml $(tag) == 'package-2019-06'
+input-file:
+  - Microsoft.StorageSync/stable/2019-06-01/storagesync.json
+```
 ### Tag: package-2019-03-01
 
 These settings apply only when `--tag=package-2019-03-01` is specified on the command line.
 
-```yaml $(tag) == 'package-2019-03-01'
+``` yaml $(tag) == 'package-2019-03-01'
```

Comment:
No need to add a space.

Reply:
I didn't modify this myself (likely code generation made the changes). I can fix it anyways.